docs/_includes/common/tooltip.html

```diff
@@ -157,12 +157,12 @@ <h3 class="infotip__heading">Infotip</h3>
                 <use xlink:href="#icon-information"></use>
             </svg>
         </button>
-        <span class="infotip__overlay">
+        <span class="infotip__overlay" style="top: 32px; left: -12px">
```

Comment:
This should be handled completely by CSS. We should not need to specify where the overlay is positioned within these examples.

Reply:
Some of them were already there in other examples but missing in some examples. They were causing weird style alignments. 
Also some of them were in the rendering portion but not in the example code.

Reply:
Let's chat offline so I'm we can go in more depth on this.

Reply:
Per discussion, please use the [computed style rules from Core](https://github.com/eBay/ebayui-core/blob/master/src/components/components/ebay-tooltip-overlay/constants.js).